README.md

```diff
@@ -13,8 +13,8 @@ Most popular tech org blog with latest technology update.
 * [Digital Ocean Engineering Blog](https://blog.digitalocean.com/tag/engineering/)
 * [Docker Engineering Blog](https://blog.docker.com/category/engineering/)
 * [Dropbox Tech Blog](https://tech.dropbox.com/)
-* [Engineering at Quora](http://engineering.quora.com/)
```

Comment:
Why did you remove this blog??

Reply:
Duplicated on line 40.

Reply:
Oh' Sorry I just missed.
Thank you so much for pointing out.
:1st_place_medal: